rpc/rpcserver/server.go

```diff
@@ -1172,16 +1172,10 @@ func StartTicketBuyerService(server *grpc.Server, loader *loader.Loader, tbCfg *
 func (t *ticketbuyerServer) StartAutoBuyer(ctx context.Context, req *pb.StartAutoBuyerRequest) (
 	*pb.StartAutoBuyerResponse, error) {
 
-	defer zero.Bytes(req.Passphrase)
-
 	wallet, ok := t.loader.LoadedWallet()
 	if !ok {
 		return nil, grpc.Errorf(codes.FailedPrecondition, "Wallet has not been loaded")
 	}
-	err := wallet.Unlock(req.Passphrase, nil)
```

Comment:
to comply with the documented api, we still need to verify that the passphrase is correct before continuing.